app/lib/search/index_simple.dart

```diff
@@ -186,8 +186,13 @@ class SimplePackageIndex implements PackageIndex {
     if (query.parsedQuery.emails.isNotEmpty) {
       packages.removeWhere((package) {
         final doc = _packages[package];
+        if (doc?.emails == null) {
```

Comment:
`doc?.emails is List<String>` might be better?

Or we could do: `doc?.emails?.contains == null`

Reply:
Why would that be different?

Reply:
It might not... only if `doc.emails` contain a non-list.. maybe? still that seems unlikely..